### PR TITLE
Pin deploy to commit

### PR DESCRIPTION
**Pull request by dannyroberts**
instead of tagging it at the beginning. This results in only tagging successful deploys, instead of tagging any attempted deploy or really any fab command.

### CODE REVIEW

**Comment by dannyroberts** [comment]
@calellowitz cc: @kaapstorm

**Comment by dannyroberts** [comment]
Also thanks @esoergel for your help orienting me

**Comment by dannyroberts** [comment]
and so far it's run fine on staging

File: fab/fabfile.py [hunk 1]
```diff
@@ -111,24 +111,26 @@ def __init__(self, env):
         self.env = env
         self.timestamp = datetime.datetime.utcnow().strftime('%Y-%m-%d_%H.%M')
         self._deploy_ref = None
+        self._deploy_tag = None
+        self._git = sh.git.bake(_tty_out=False)
```
Comment (esoergel) [inline fab/fabfile.py]: :+1:

**Comment by esoergel** [comment]
Looks great!  Waiting on build.  Although fabric stuff probably has no effect on travis, huh?

**Comment by dannyroberts** [comment]
:white_check_mark:

**Pull request merged**